src/bl-error/bl-error-handler.ts

```diff
@@ -133,7 +133,8 @@ export class BlErrorHandler {
         blapiErrorResponse.msg = "server error";
         break;
       case 801:
-        blapiErrorResponse.msg = "blid already handed out";
+        blapiErrorResponse.msg =
+          "En eller flere av bøkene du prøver å dele ut er allerede aktiv på en annen kunde. Prøv å dele ut én og én bok for å finne synderen.";
```

Comment:
I know you quoted Bjørgen directly, but we _cannot_ use the word "synderen" 💩  
```suggestion
          "En eller flere av bøkene du prøver å dele ut er allerede aktiv på en annen kunde. Prøv å dele ut én og én bok for å finne ut hvilke bøker dette gjelder.";
```